src/qt/governancelist.cpp

```diff
@@ -417,6 +421,34 @@ void GovernanceList::updateProposalCount() const
     ui->countLabel->setText(QString::number(proposalModelProxy->rowCount()));
 }
 
+void GovernanceList::showCreateProposalDialog()
+{
+    if (!this->clientModel || !this->walletModel) {
+        QMessageBox::warning(this, tr("Unavailable"), tr("A synced node and an unlocked wallet are required."));
+        return;
+    }
+    if (!proposalWizard) {
+        proposalWizard = new ProposalWizard(this->clientModel->node(), this->walletModel, this);
```

Comment:
I see ProposalWizard is created with `new` but it is nowhere to be removed. Consider using unique_ptr here